spec/features/movies/show_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "rails_helper"
+
+RSpec.describe "Show" do 
+  before :each do 
+    @pixar = Studio.create!(name: "Pixar Studios", location: "Hollywood")
+    @toy_1 = Movie.create!(title: "Toy Story", creation_year: "1995", genre: "Animation", studio_id: @pixar.id)
+    
+    @hanks = Actor.create!(name: "Tom Hanks", age: 42)
+    @allen = Actor.create!(name: "Tim Allen", age: 41)
+    @reeves = Actor.create!(name: "Keanu Reeves", age: 40)
+
+    MovieActor.create!(movie_id: @toy_1.id, actor_id: @hanks.id)
+    MovieActor.create!(movie_id: @toy_1.id, actor_id: @allen.id)
+    MovieActor.create!(movie_id: @toy_1.id, actor_id: @reeves.id)
+  end
+
+  it "shows a movies attributes and their actors" do 
+    visit "/movies/#{@toy_1.id}"
+
+    expect(page).to have_content("Movie Show Page")
+    expect(page).to have_content(@toy_1.title)
+    expect(page).to have_content(@toy_1.creation_year)
+    expect(page).to have_content(@toy_1.genre)
+
+    expect(page).to have_content(@hanks.name)
+    expect(page).to have_content(@hanks.age)
+    expect(page).to have_content(@allen.name)
+    expect(page).to have_content(@allen.age)
+    expect(page).to have_content(@reeves.name)
+    expect(page).to have_content(@reeves.age)
+
+    expect(@reeves.name).to appear_before(@allen.name)
+    expect(@allen.name).to appear_before(@hanks.name)
+
+    expect(page).to have_content("Actors Average Age:")
+    expect(Actor.average_age).to eq(41.0)
```

Comment:
When we're testing this functionality, we don't want to call the production code that we're testing. In this case, we're testing the `average_age` method, so it'd be better to expect something like
`expect(page).to have_content("Actors Average Age: 41")` rather than separating these two assertions and calling the model method in the assertion

Reply:
We want to do the calculation and know what to expect, and then by asserting this expected value (41), we are verifying that the production code under the hood is running this calculation correctly